packages/app/src/components/DimensionsPanel/Dialogs/AddToLayoutButton/AddToLayoutButton.js

```diff
@@ -21,6 +25,8 @@ export class AddToLayoutButton extends Component {
 
     state = { anchorEl: null }
 
+    labelPrefix = i18n.t('Add to')
```

Comment:
`Add to` might not translate well, it's better to include the interpolation in-line (`Add to {{axisName}}`), maybe we should do that in the two places this is used instead of adding an instance variable?

Reply:
As an example - in some languages, the object might come before the verb.

Reply:
(Yes, I realize this is [very rare](https://en.wikipedia.org/wiki/Object%E2%80%93verb%E2%80%93subject) but there may be different ways to structure imperatives in different languages)

Reply:
Indeed, i18n strings should retain as much context as possible. Fixed